### DISCUSSION TITLE
Filter not working in local instance

### DISCUSSION THREAD

**Discussion by Tulasi08** [broadinstitute/seqr#4075] [Q&A]
Hi Team,

We recently noticed that in our local instance, when we apply filters like High Quality variants (AB > 25) we are getting variants which are less than 25% as well?
![image](https://github.com/broadinstitute/seqr/assets/122073339/51f41fec-3078-4720-bf4c-9080d6dda958)

![image](https://github.com/broadinstitute/seqr/assets/122073339/e355785a-47d9-41d3-9da1-d11ba517c359)

![image](https://github.com/broadinstitute/seqr/assets/122073339/9b6dea11-d0eb-431a-a3fa-43998e92c2b9)

Similarly, For Denovo, all and recessive inheritance the Pathogenic/Likely Pathogenic filter is not working and we also get VUS/Benign/Likely Benign variants. It also picks up missense, in frame variants, even manually after de-selecting them. Please refer to the screenshot for your reference.
![image](https://github.com/broadinstitute/seqr/assets/122073339/e87e7049-eeda-4fd9-98d9-6026c5b83f49)

Can we know why is it so and if this is the expected behaviour if multiple filters are applied? Also wanted to clarify if the OR logic is followed or AND logic is followed while the filters are applied. If there is an option to apply AND filter how should we do it?

Looking forward to hear from you.

Thanks and Regards,
Tulasi S

**Comment by hanars** [comment]
Hi Tulasi,

As a procedural note, please submit your questions about seqr to the github discussion forum, not as bug tickets. I can create bug tickets if there are actual code changes that are needed, but in general when you are asking questions about seqr's behavior or how to set up a local installation there is no associated engineering work for our team, which means they do not warrant a ticket. Moreover, discussion posts are much more usable to the broader seqr community, and it is generally helpful to have these discussions shared with other users who may have the same questions. I have converted your past couple issues to discussions, but would appreciate in the future if you could submit them to the discussion in the first place.

Regarding your question, if you have applied a "pathogenicity" filter in your search then clinvar pathogenic variants override most of the filters, including quality, annotations, and frequency filters, so this behavior is expected.  
<img width="1304" alt="Screenshot 2024-05-09 at 11 09 40 AM" src="https://github.com/broadinstitute/seqr/assets/24598672/bdcc1f03-4c30-4267-ab15-30c11ad5a904">

**Comment by Tulasi08** [comment]
Hi Hana,

Noted on this. However, if that is the case, then for annotation even if missense may come up for P/LP/VUS etc, not sure why benign and likely benign (synonymous, in frame) also came up, that shouldn't come right? 

Thanks and Regards,
Tulasi S

**Reply by hanars** [reply]
The variants you showed in your search results are pathogenic in clinvar and should override quality filters. If you are seeing variants you do not expect in search results, you would need to provide screenshots of the exact search criteria you applied and the unexpected variants being returned, as I can not answer theoretical questions about why a variant is coming back in search without all the details.
